cob_obstacle_distance/include/cob_obstacle_distance/marker_shapes.hpp

```diff
@@ -52,30 +51,30 @@ class MarkerShape : public IMarkerShape
         FclMarkerConverter<T> fcl_marker_converter_;
         visualization_msgs::Marker marker_;
 
-        void init(double x, double y, double z,
+        void init(const std::string root_frame, double x, double y, double z,
```

Comment:
see this [comment](https://github.com/ipa-fxm/cob_control/pull/7/files#r34412108)